torch/csrc/autograd/variable.cpp

```diff
@@ -91,6 +93,13 @@ namespace impl {
     }
   }
 
+  void disable_rebase_history(Variable& self) {
+    TORCH_CHECK(self.is_view(), "disable_rebase_history can only be called on views.");
```

Comment:
Is this a CHECK or an ASSERT? This doesn't seem like public API surface to me

Reply:
That would be and assert. I'll change it to `TORCH_INTERNAL_ASSERT` as well.